dags/main_summary.py

```diff
@@ -135,34 +135,28 @@
 
 register_status(main_summary, "Main Summary", "A summary view of main pings.")
 
-addons = EMRSparkOperator(
+addons = bigquery_etl_query(
     task_id="addons",
-    job_name="Addons View",
-    execution_timeout=timedelta(hours=4),
-    instance_count=3,
-    env=tbv_envvar("com.mozilla.telemetry.views.AddonsView", {
-        "from": "{{ ds_nodash }}",
-        "to": "{{ ds_nodash }}",
-        "bucket": "{{ task.__class__.private_output_bucket }}"}),
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/telemetry_batch_view.py",
+    destination_table="addons_v2",
+    dataset_id="telemetry_derived",
     dag=dag)
 
-addons_bigquery_load = SubDagOperator(
-    subdag=load_to_bigquery(
+addons_export = SubDagOperator(
+    subdag=export_to_parquet(
+        table="moz-fx-data-derived-datasets:telemetry_derived.addons_v2${{ds_nodash}}",
+        static_partitions="submission_date_s3={{ds_nodash}}",
+        arguments=[
+            "--partition-by=sample_id",
+            "--drop=submission_date",
+            # no bigint columns, convert them all to int for backward compatibility
+            "--bigint-columns=",
```

Comment:
Is there a particular reason that this option is here, but not in addon_aggregates_v2? This is also missing a value, does this default to casting all bigint columns into int if the value is None?

Reply:
> Is there a particular reason that this option is here, but not in addon_aggregates_v2?

in `addon_aggregates_v2` there are no int columns, only bigint, so we don't need this.

> This is also missing a value

it's an empty list of values, indicating that all bigint columns should be cast to int.